src/app/static/src/app/store/load/actions.ts

```diff
@@ -74,8 +78,11 @@ export const actions: ActionTree<LoadState, RootState> & LoadActions = {
 
         if (!rootGetters.isGuest) {
             await (dispatch("projects/createProject", {name: state.projectName || null}, {root: true}));
-            savedState.projects.currentProject = rootState.projects.currentProject;
-            savedState.projects.currentVersion = rootState.projects.currentVersion;
```

Comment:
This was causing rehydrate from JSON to fail as `savedState.projects` was undefined. Looking at the state we save out this must have been removed at some point. I've initialised the state here like we do for project rehydrate

Reply:
I guess we can revert this when we take out the rehydrate from JSON, since we should always be starting from a default project state?

Reply:
Yep that's right, will remove that in next PR